src/presets.js

```diff
@@ -1125,6 +1125,45 @@ module.exports = {
 		},
 	},
 
+	updateContract: {
+		/**
+		 * @summary Update contract version
+		 * @function
+		 * @public
+		 *
+		 * @param {Object} options - options
+		 * @param {Boolean|RegExp} [options.clean=true] - determines how to sanitise the version
+		 * @param {String} cwd - current working directory
+		 * @param {String} version - version
+		 * @param {Function} callback - callback (error)
+		 * @returns {null}
+		 *
+		 * @example
+		 * presets.updateContract.version({}, process.cwd(), '1.0.0', (error) => {
+		 *   if (error) {
+		 *     throw error;
+		 *   }
+		 * });
+		 */
+		version: (options, cwd, version, callback) => {
+			const cleanFn = getCleanFunction(options);
+			const cleanedVersion = cleanFn(version);
+
+			if (!cleanedVersion) {
+				return callback(new Error(`Invalid version: ${version}`));
+			}
+
+			const contract = path.join(cwd, 'balena.yml');
+			if (!fs.existsSync(contract)) {
+				return callback(null, version);
+			}
+
+			const content = yaml.safeLoad(fs.readFileSync(contract, 'utf8'));
+			content['version'] = cleanedVersion;
```

Comment:
NIT: this is usually more idiomatic as `content.version = cleanedVersion`, object access with square brackets is usually only used in the context of a variable key, i.e.
```
const key = 'version'
content[key] = cleanedVersion
```

Reply:
Agreed, done.